CONTRIBUTING.md

```diff
@@ -479,6 +481,40 @@ run as part of the regular build, since they have some particular requirements.
 See the [CLI CONTRIBUTING.md file](packages/aws-cdk/CONTRIBUTING.md) for
 more information on running those tests.
 
+### API Compatibility Checks
+
+All stable APIs in the CDK go through a compatibility check during build using
+the [jsii-diff] tool. This tool downloads the latest released version from npm
+and verifies that the APIs in the current build have not changed in a breaking
+way.
+
+[jsii-diff]: https://www.npmjs.com/package/jsii-diff
+
+Compatibility checks always run as part of a full build (`build.sh`).
+
+You can use `yarn compat` to run compatibility checks for all modules:
```

Comment:
This requires everything to have been built to jsii assemblies, might want to work that in there.

Reply:
ok added "yarn build" before